lib/v2/uber/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'uber.com': {
+        _name: 'Uber Blog',
+        www: [
+            {
+                title: 'Uber Blog - Engineering',
+                docs: 'https://docs.rsshub.app/blog.html#you-bu',
```

Comment:
I'm not sure whether this doc link is correct

Reply:
I have made some updates according to the comments. Let me known if mistakes exist. Thanks again.